src/main/java/org/codewars/utils/TaskRunner.java

```diff
@@ -82,8 +88,16 @@ public static double runTwoDecimalPlaces(){
         return 0.0;
     }
 
-    public static boolean runAmIWilson(){
-        return false;
+    public static void runAmIWilson()
```

Comment:
bc we need user which TaskRunner'll get by constructer, this method can't be static (but maby we'll change this idea on today meeting)